### DISCUSSION TITLE
Move Items to another mailbox

### DISCUSSION THREAD

**Discussion by gismcieri** [ecederstrand/exchangelib#980] [Q&A] [answered]
Hi,

I was wondering if it was possible to move emails to another users email box without forwarding.

I currently have the item and can move it around its own mail box but i tried moving it to another account and it can't find the folder.

FYI i know i can forward the email but I want to move it to preserve the metadata.

Any thoughts?

Thanks,
Matt

**Comment by ecederstrand** [comment, marked as answer]
A quick googling suggests that it is possible to move an item to another mailbox, provided you have permissions to both mailboxes.

Something like this:
```python
account = Account("some_user@example.com", credentials=...)
message = account.inbox.all()[0]
message.move(to_folder=DistinguishedFolderId(
    id=Calendar.DISTINGUISHED_FOLDER_ID,
    mailbox=Mailbox(email_address="other_user@example.com")
))
```

Another possibility is to use the export/upload functionality: https://ecederstrand.github.io/exchangelib/#export-and-upload

**Reply by gismcieri** [reply]
I have access but its another account.  so made account2 and tried that as the 2 folder.  This like it might have to be an account with access to another account.

**Reply by gismcieri** [reply]
Thanks for the response.  I figured out what i was doing wrong.  I had access to the other account but the user that was moving the message didn't so even though i was logged into both accounts i couldn't move it.  You still have to have the sharing permission set up.
